packages/client/app/profile/UserSettings/UserSettings.tsx

```diff
@@ -0,0 +1,109 @@
+import { FC, useContext } from "react";
+import { useTheme, Theme } from '@mui/material/styles';
+import { AuthContext } from "@/context/AuthContext";
+import { Typography, Box, Input, Button } from '@mui/material';
+import LinearProgress from '@mui/material/LinearProgress';
+import { useFormik } from 'formik';
+import { profileSettingsValidation } from "@/utils/validation/profileSettingsValidation";
```

Comment:
for some reason eslint is not working...b/c I can see wrong formatting

try `npm run lint`